CovidTracker/Pages/Index.cshtml

```diff
@@ -52,12 +52,12 @@
     <div class="row">
         <div class="col-md-6">
             <div class="card shadow border-0">
-                <img src="/img/SaferChoices.jpg" style="height: 650px" alt="Safer Choices">
```

Comment:
Added the img_style class to the image - Safer choices (image 3 in index page)

Reply:
Good idea.  Avoid using inline styles in XHTML 1.0 strict.  Use linked stylesheets instead.